tequilapi/endpoints/health_check.go

```diff
@@ -25,16 +25,27 @@ import (
 	"time"
 )
 
+// swagger:model
 type healthCheckData struct {
-	Uptime    string    `json:"uptime"`
-	Process   int       `json:"process"`
+	// example: 25h53m33.540493171s
+	Uptime string `json:"uptime"`
+
+	// example: 10449
+	Process int `json:"process"`
+
+	// example: 0.0.6
 	Version   string    `json:"version"`
 	BuildInfo buildInfo `json:"buildInfo"`
 }
 
 type buildInfo struct {
-	Commit      string `json:"commit"`
-	Branch      string `json:"branch"`
+	// example: <unknown>
```

Comment:
Why this struct doesn't have "// swagger:model"  heading?

Reply:
added missing meta tags